examples/interop-examples/README.md

```diff
@@ -0,0 +1,102 @@
+# interop examples
+
+**WIP**
+
+## Run
+```
+cd examples/interop-examples
+npm i
+npm start
+```
+
+Open http://127.0.0.1:8080 in your favorite browser.
+
+## Tutorial
+
+Steps
+1. create IPFS instance
+
+First, we need an IPFS instance that we'll use throughout the tutorial. For the sake of convinience, we'll use `ipfs-daemon` module to create and IPFS instance.
+
+Add `ipfs-daemon` to your project:
+```
+npm i ipfs-daemon --save
+```
+
+Add `ipfs-daemon` to your html:
+```html
+<script type="text/javascript" src="node_modules/ipfs-daemon/dist/ipfs-browser-daemon.js" charset="utf-8"></script>
+```
+
+Create the IPFS instance and wait for it to start:
+
+```javascript
+const ipfs = new IpfsDaemon({
+  // Directory to which save IPFS data to
+  IpfsDataDir: '/path/to/ipfs/data',
+  // IPFS dev server: webrtc-star-signalling.cloud.ipfs.team
+  SignalServer: '188.166.203.82:20000',
+})
+
+ipfs.on('error', (e) => console.error(e))
+ipfs.on('ready', () => {
+  // We're ready to use IPFS!
+})
+```
+
+2. listen for file drop
```

Comment:
The goal of this demo is to "access files from go-ipfs", and not adding files inside the browser.